ros/src/actuation/vehicles/packages/as/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(as)
+
+find_package(
+  catkin REQUIRED COMPONENTS
+  roscpp
+  std_msgs
+  geometry_msgs
+  module_comm_msgs
```

Comment:
This commit contains code that depends on "module_comm_msgs".
But "module_comm_msgs" does not exist.